### DISCUSSION TITLE
Collect more content that can be used to demonstrate Mirador 3 features

### DISCUSSION THREAD

**Discussion by mejackreed** [ProjectMirador/mirador#3463] [Q&A]
We are looking for manifests in both IIIF Presentation v2 and v3 for the following types of content.

- viewing hints of facing-pages or non-paged
- viewing hint of continuous
- top-to-bottom (or BTT) but not continuous
- multiple sequences
- IIIF v3 equivalents of the above

Please anyone who may have this type of content, file comments below.

**Comment by rentonsa** [comment]
Hi Jack, we need to do a bit more on this old thing (annotations, navigation etc)- but here it is in its current state: 
https://librarylabs.ed.ac.uk/iiif/manifest/mahabharataFinal.json
The naming convention is very much non-standard as it was such a one-off.

**Comment by mejackreed** [comment]
https://iiifpres.bdrc.io/vo:bdr:I0886/manifest see https://github.com/ProjectMirador/mirador/issues/1957#issuecomment-631081983

**Comment by regisrob** [comment]
This Manifest from e-codices is the only one having multiple sequences I know of: https://www.e-codices.unifr.ch/metadata/iiif/gau-Fragment/manifest.json

This one from the Vatican has a `non-paged` hint for some canvases (https://digi.vatlib.it/iiif/MSS_Vat.sir.19/manifest.json), but at the same time the manifest and sequence have a `individuals` hint, so I guess it should be ignored anyway...

**Comment by MImranAsghar** [comment]
Hi jack, here are a couple examples of multiple – sequences 

https://gist.github.com/MImranAsghar/3ccef5a7e5aa856d5e0f0eb9c7670d08
https://gist.github.com/MImranAsghar/11310428b56bf917e534940bfb9804af
https://gist.github.com/MImranAsghar/01409fb020a8c3571ff8310ac53c85e1
https://gist.github.com/MImranAsghar/246a59b61781b00288acb1a224f14552

**Comment by triplingual** [comment]
v3 recipes that present some of the desired features:
TTB, not continuous: https://preview.iiif.io/cookbook/0010-book-2-viewing-direction/recipe/0010-book-2-viewing-direction/ ([relevant manifest in Mirador](https://projectmirador.org/embed/?iiif-content=https://preview.iiif.io/cookbook/0010-book-2-viewing-direction/recipe/0010-book-2-viewing-direction/manifest-ttb.json))
LTR, continuous: https://preview.iiif.io/cookbook/0011-book-3-behavior/recipe/0011-book-3-behavior/ ([relevant manifest in Mirador](https://projectmirador.org/embed/?iiif-content=https://preview.iiif.io/cookbook/0011-book-3-behavior/recipe/0011-book-3-behavior/manifest-continuous.json))
